scripts/fix_uuids.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# This script finds empty @id: and replace with new uuids
+# Then it finds for duplicates and replaces last occurrence with new uuids
+# finally it fixes '@id:xyz' to '@id: xyz' (missing space after :)
+
+# finds occurrences of empty @id: testimony tags
+EMPTY_IDS=$(grep -E -i -r -n "@id:(.+[[:blank:]]|$)" tests/foreman/ --include=*.py)
+
+if [ -n "$EMPTY_IDS" ]; then
+   echo "Generating new UUIDS for empty @id tags..."
+else
+   echo "No empty @id was found"
+fi
+
+# iterate if any empty @id found
+for output_line in $EMPTY_IDS
+do
+    if (echo "$output_line" | grep "tests/foreman"); then
```

Comment:
I needed to include it here because in some cases grep output have been wrapped, but I can try different approach.